app/src/main/java/io/apicurio/registry/storage/impl/sql/mappers/RuleConfigurationDtoMapper.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2021 Red Hat
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.apicurio.registry.storage.impl.sql.mappers;
+
+import java.sql.ResultSet;
+import java.sql.SQLException;
+
+import io.apicurio.registry.storage.dto.RuleConfigurationDto;
+import io.apicurio.registry.storage.impl.sql.RowMapper;
+
+/**
+ * @author eric.wittmann@gmail.com
+ */
+public class RuleConfigurationDtoMapper implements RowMapper<RuleConfigurationDto> {
+
+    public static final RuleConfigurationDtoMapper instance = new RuleConfigurationDtoMapper();
+
+    /**
+     * Constructor.
+     */
+    private RuleConfigurationDtoMapper() {
+    }
+
+    /**
+     * @see org.jdbi.v3.core.mapper.RowMapper#map(java.sql.ResultSet, org.jdbi.v3.core.statement.StatementContext)
```

Comment:
I've missed this in my previous review, I don't think it's related to the compilation issues but we should remove the reference to jdbi.

Reply:
I've updated these, thanks.